scripts/examples/learner_example_rag.py

```diff
@@ -0,0 +1,34 @@
+import os
+from pathlib import Path
+from dotenv import find_dotenv, load_dotenv
+from huggingface_hub import login
+
+from ontolearner.learner_pipeline import LearnerPipeline
+from ontolearner.ontology import Wine
+from ontolearner.utils.train_test_split import train_test_split
+
+_ = load_dotenv(find_dotenv())
```

Comment:
Please simply add `token=" "` (empty string), you already explained in the documentation that this is up to users how to specify the token!